File: CRUDAPI/CRUDAPI/Models/Tarefas.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CRUDAPI.Models
{
    public class Tarefas
    {
        [Key]
        public int TarefaId { get; set; }
        public string NomeTarefa { get; set; }
        public string Descricao { get; set; }



        [ForeignKey("PessoaId")]
        public int PessoaId { get; set; }

        public Pessoas Pessoa { get; set; }

    }
}

File: src/app/Tarefas.ts
export class Tarefas {
  tarefaId: number = 0;
  nomeTarefa: string = '';
  descricao: string = '';
  pessoaId: number = 0;
  pessoa: Pessoa | null = null;
}

export class Pessoa {
  pessoaId: number = 0;
  nome: string = '';
  sobrenome: string = '';
  idade: number | null = null;
  profissao: string = '';
}
